fetch-utils.js

```diff
@@ -37,6 +37,29 @@ export async function logout() {
     return (window.location.href = '../');
 }
 
-// function checkError({ data, error }) {
-//     return error ? console.error(error) : data;
-// }
+//-------------------My Stuff-----------------
+
+export async function getWorkshops() {
+    // fetch all families and their bunnies
+    const response = await client.from('workshops').select('*, clients(*)').match({ 'clients.user_id': client.auth.session().user.id });
+    //console.log(client.auth.session().user.id);
+    return checkError(response);
+}
+
```

Comment:
Excellent job making your users match their information to make it user specific!